### PR TITLE
[orc8r][docs][helm] Minikube documentation

### PR DESCRIPTION
**Pull request by Scott8440**
Signed-off-by: Scott8440 <scott8440@gmail.com>

## Summary
* Document minikube development process in one place with all required steps
* Fix bug in secrets chart so this deployment actually works

## Test Plan
Docs generated:
image

Follow documented process exactly, orc8r deployed and API works:
![image](https://user-images.githubusercontent.com/13274915/96167881-dbe4e980-0ed4-11eb-8c6a-f015bd1ff3ef.png

### CODE REVIEW

File: docs/docusaurus/i18n/en.json [hunk 1]
```diff
@@ -642,6 +645,37 @@
...
+        "title": "Install Orchestrator"
+      },
+      "version-1.3.0/orc8r/version-1.3.0-upgrade_1_3": {
+        "title": "Upgrade to v1.3"
+      },
+      "version-1.3.0/orc8r/version-1.3.0-upgrade_intro": {
+        "title": "Orchestrator Upgrades: Introduction"
+      },
+      "version-1.3.0/proposals/version-1.3.0-p004_fua_restrict_feature": {
+        "title": "Service Restriction Feature"
```
Comment (xjtian) [inline docs/docusaurus/i18n/en.json:678]: what's going on here

File: docs/docusaurus/sidebars.json [hunk 2]
```diff
@@ -78,6 +78,9 @@
       "proposals/p001_vpn_config_from_api",
       "proposals/p002_scaled_prometheus_pipeline",
       "proposals/p003_qos_enforcement"
+    ],
+    "Development Guide": [
+      "development/minikube_deployment"
```
Comment (xjtian) [inline docs/docusaurus/sidebars.json]: make this a subsection under Orchestrator "Development"

File: docs/readmes/development/minikube_deployment.md [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+$ for image in controller nginx ; do ${PUBLISH} -r ${DOCKER_REGISTRY} -i ${image} -v ${MAGMA_TAG} ; done
+```
+
+Now do the same for the NMS images (if you want to deploy NMS)
+
+```
+cd MAGMA_ROOT/nms/app/packages/magmalte
+docker-compose build magmalte
+${PUBLISH} -r ${REGISTRY} -i magmalte -v ${MAGMA_TAG}
+```
```
Comment (xjtian) [inline docs/readmes/development/minikube_deployment.md]: I think you can just link to the relevant section of the deploy docs for all this "Build Orchestrator". Then if we change this procedure we can just update it in one place

File: docs/readmes/development/minikube_deployment.md [hunk 4]
```diff
@@ -0,0 +1,356 @@
...
+    --set=docker.password=$DOCKER_PASSWORD |
+    kubectl apply -f -
+```
+
+### Create Values File
+
+Here is a minimum values file that can be used to deploy orc8r (tested with 1.4.39).
+Make sure to replace `<DOCKER_REGISTRY>` with your registry and `<TAG>` with your tag.
+
+```
```
Comment (xjtian) [inline docs/readmes/development/minikube_deployment.md]: you wanna put this somewhere in the repo so it's tracked as code? `orc8r/cloud/helm/orc8r/examples` maybe? Or just under `orc8r/cloud/helm`?

File: docs/readmes/development/minikube_deployment.md [hunk 5]
```diff
@@ -0,0 +1,356 @@
...
+
+### Access NMS
+
+Create org and user in NMS:
+
+```
+kubectl exec -n magma \
+  $(kubectl get pod -n magma -l app.kubernetes.io/component=magmalte -o jsonpath="{.items[0].metadata.name}") -- \
+  yarn setAdminPassword magma-test admin@magma.test password1234
+```
```
Comment (xjtian) [inline docs/readmes/development/minikube_deployment.md]: x-ref the corresponding portion of the deployment docs for these sections as well, and just keep the nginx port-forwarding part in this document

**Review by xjtian — CHANGES_REQUESTED** [review]

File: docs/docusaurus/i18n/en.json (hunk 1 above)
Comment (Scott8440) [reply docs/docusaurus/i18n/en.json:678]: whoops

Comment (Scott8440) [reply docs/docusaurus/i18n/en.json:678]: Ok actually this gets generated every time I run `yarn build`. I think last time someone did this it just wasn't build so these generated changes weren't produced.

**Comment by Scott8440** [comment]
Address comments. Move values file to code, link to existing documents.

**Review by xjtian — APPROVED** [review]

**Pull request merged**